lib/components/normal-components/Panel.ts

```diff
@@ -145,7 +148,7 @@ export class Panel extends Group<typeof panelProps> {
       if (isFinite(minX)) {
         const boundsWidth = maxX - minX
         const boundsHeight = maxY - minY
-        const margin = TAB_CONFIG.TAB_DEPTH * 3
+        const margin = 5
```

Comment:
i guess this will need to be a parameter? we shouldn't have a magic number- maybe make this DEFAULT_TAB_DEPTH